src/scss/styles.scss

```diff
@@ -0,0 +1,263 @@
+@import "./variables/variables";
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
+@import "./components/buttons";
+
+/* Global */
+body {
+  background-color: #14063d;
+}
+
+html,
+body {
```

Comment:
This could be done using bootstrap. You can use lh-1 and h-100 classes. In fact, a lot of the style in the file could be done using pure bootstrap.

Reply:
Thank you very much, try to fix.